src/python/ensembl/io/genomio/genome_metadata/prepare.py

```diff
@@ -232,6 +233,7 @@ def prepare_genome_metadata(
     output_file: PathLike,
     gff3_file: Optional[PathLike] = None,
     base_api_url: str = DEFAULT_API_URL,
+    mock_run: bool = True,
```

Comment:
```suggestion
    mock_run: bool = False,
```
I would set this as `False` by default, since I believe we want to this to be the exception, not the norm.

Reply:
Yes, that makes sense